### PR TITLE
Completed phase 1 react-design

### PR DESCRIPTION
**Pull request by amorabryan**
image

### CODE REVIEW

**Comment by amorabryan** [comment]
image

**Comment by amorabryan** [comment]
Recording 2023-08-09 at 14 08 05

File: react-design/react-design-notes.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# react-design-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What are the phases in the React design process?
```
Comment (robgmerrill) [inline react-design/react-design-notes.md:7]: This question is referring to the three phases we walked through in the exercise.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**